FRW/Services/Manager/LocalUserDefaults.swift

```diff
@@ -52,59 +52,10 @@ extension LocalUserDefaults {
 
         case customToken
     }
-
-    enum FlowNetworkType: String, CaseIterable, Codable {
```

Comment:
I moved this enum out of the `LocalUserDefaults` namespace and to its own `FlowNetworkType.swift` file